src/fs/metadata_log.cc

```diff
@@ -255,6 +334,7 @@ metadata_log::flush_result metadata_log::schedule_flush_of_curr_cluster_and_chan
 
     auto append_res = _curr_cluster_buff->append(ondisk_next_metadata_cluster {*next_cluster});
     assert(append_res == metadata_to_disk_buffer::APPENDED);
+    _log_cluster_count++;
```

Comment:
```suggestion
    ++_log_cluster_count;
```
IMO this looks better, but I will respect if you prefer otherwise.

Reply:
Please either apply this or resolve this.